geopandas/array.py

```diff
@@ -1044,25 +1044,30 @@ def fillna(self, value=None, method=None, limit=None):
             Alternatively, an GeometryArray 'value' can be given. It's expected
             that the GeometryArray has the same length as 'self'.
 
-        method : {'backfill', 'bfill', 'pad', 'ffill', None}, default None
-            Method to use for filling holes in reindexed Series
-            pad / ffill: propagate last valid observation forward to next valid
-            backfill / bfill: use NEXT valid observation to fill gap
+        method : {'backfill', 'bfill', 'ffill', None}, default None
+            Method to use for filling holes in reindexed GeoSeries:
```

Comment:
```suggestion
            Method to use for filling holes:
```

(the "reindexed" is a bit confusing, since this method is not reindexing and just filling NAs. I know this comes from pandas, but I think it should be fixed there as well)